front-end/src/app/shared/services/login.service.ts

```
@@ -53,6 +53,6 @@ export class LoginService extends DestroyerComponent {
   }
 
   public userIsAuthenticated() {
-    return new Date() < new Date(parseInt(this.cookieService.get('ffapi_timeout')) * 1000);
+    return new Date() < new Date(parseInt(this.cookieService.get(environment.ffapiTimeoutCookieName)) * 1000);
```

similar to API comment, we may want to add a default cookie name here in case the environment var isn't set.  Otherwise, parseInt may fail and break all api requests in the guard.

---

I think this is one where failing loudly on missing the configuration is what we want.

We should add a check, though that throws a descriptive error like "api timeout cookie name not defined"

---

In the case of the web app, it will not compile if the cookie name definition is missing.  I have added a check for it, and a check for whether it is defined, for good measure, though.